gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.4-bin.zip
```

Comment:
Why do we choose version 7.6.4 and not 8.6?

Reply:
There are a lot of changes when upgrading from version 7 to 8. Should it be 8.6 instead?
https://docs.gradle.org/current/userguide/upgrading_version_7.html#changes_8.0